overlord/auth/auth.go

```diff
@@ -180,18 +180,18 @@ func RemoveUser(st *state.State, userID int) error {
 	return fmt.Errorf("invalid user")
 }
 
-func UserCount(st *state.State) (int, error) {
+func Users(st *state.State) ([]UserState, error) {
```

Comment:
`[]*UserState`, similar to the client point.

Also, can we please have a test for this one?